pre_commit/main.py

```diff
@@ -148,7 +148,7 @@ def _adjust_args_and_chdir(args):
 def main(argv=None):
     argv = argv if argv is not None else sys.argv[1:]
     argv = [five.to_text(arg) for arg in argv]
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(prog='pre_commit')
```

Comment:
`pre-commit` is stylized this way
```suggestion
    parser = argparse.ArgumentParser(prog='pre-commit')
```

Reply:
Woops! Not sure where I got that from. Fixed c7d938c 👍